mkosi/installer/moss.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from collections.abc import Sequence
+
+from mkosi.bubblewrap import apivfs_cmd, bwrap
+from mkosi.context import Context
+from mkosi.types import PathString
+from mkosi.util import sort_packages
+
+
+def moss_cmd(context: Context) -> list[PathString]:
+    cmdline: list[PathString] = [
+        "moss",
+        "--directory",
+        f"{context.root}",
+        "--yes-all",
+    ]
+
+    return cmdline
+
+
+def setup_moss(
+    context: Context,
+) -> None:
+    (context.pkgmngr / "etc/moss/repo.d").mkdir(exist_ok=True, parents=True)
```

Comment:
You create `etc/moss/repo.d` in the `pkgmngr` directory but never configure `moss` to actually look for repos in that directory.

Reply:
Does `moss` look in the host's root directory or in the image's root directory for its configuration? If it's in the host's root directory then this is fine.